File: Practice/AOJ/ITP1/ITP1_5C/itp1_5c.rb
loop do
    h, w = gets.chomp.split.map(&:to_i)
    if (h + w).zero?
        break
    end
    (0...h).each do |y|
        line = ""
        (0...w).each do |x|
            line = (y + x).even? ? line + "#" : line + "."
        end
        puts line
    end
    puts
end

File: Practice/AOJ/ITP1/ITP1_5C/itp1_5c.py
while True:
    h, w = map(int, input().split())
    if w+h == 0:
        break
    for y in range(h):
        line = ""
        for x in range(w):
            if (y+x) % 2 == 0:
                line += "#"
            else:
                line += "."
        print(line)
    print()
